### ISSUE TITLE
Prepublish checklist error "Story cover too small" triggered on large images

### ISSUE THREAD

**Issue by bmattb** [GoogleForCreators/web-stories-wp#5379]
## Bug Description

Story cover image too small” shows up when using an image with dimensions 1350 by 900 pixels (large enough but wrong aspect ratio)

## Expected Behaviour

Error should read "Story cover image wrong aspect ratio"

## Steps to Reproduce

1. Go to Document tab, select a cover image
2. Select an image that is large, but not 3:4 aspect ratio
3. See that error title says "Story cover image too small" when it should be "Story cover image wrong aspect ratio"

## Screenshots

![Screen Shot 2020-11-20 at 4 11 10 PM](https://user-images.githubusercontent.com/58738530/99850484-73300300-2b4b-11eb-9ec0-75903b84417f.png)
![Screen Shot 2020-11-20 at 4 14 01 PM](https://user-images.githubusercontent.com/58738530/99850491-75925d00-2b4b-11eb-84cf-d092ad7ec9e9.png)

## Additional Context

- Plugin Version:
- Operating System:
- Browser:

---

_Do not alter or remove anything below. The following sections will be managed by moderators only._

**Comment by o-fernandez** [comment]
**Edit: Two things to do in this ticket.**

* separate out image too small check from wrong dimensions. In the text strings we have these two errors, note that the help text is the same, but the error title is different. 

**Story cover image too small** | Should be at least 640x853px and maintain a 3:4 aspect ratio.
**Story cover image wrong aspect ratio** | Should be at least 640x853px and maintain a 3:4 aspect ratio.

* separately, in another ticket, we could explore how to make it easier for the user to fix aspect ratios issues. 

I'll update the ticket to focus on the first issue.

**Comment by swissspidy** [comment]
Related / perhaps blocked by: #5229

cc @embarks

**Comment by carloskelly13** [comment]
![Screen Shot 2020-11-24 at 11.36.56 AM.png](https://images.zenhubusercontent.com/5ea09bca6a44046f869bafe0/426270c3-473b-47e8-b9ba-11734884d4b5)

![Screen Shot 2020-11-24 at 11.38.08 AM.png](https://images.zenhubusercontent.com/5ea09bca6a44046f869bafe0/f7003c35-061f-4438-8ac0-dd0a12d85d2a)

This one is going to be a bit tricky to fix. On cover story set we get the wrong dimensions inside the story data. This is corrected when the story is reloaded.

And this is coming directly from the story data.

cc @swissspidy

**Comment by swissspidy** [comment]
@carlos-kelly Yeah this was noted in #5229

> Note: When updating the cover image, the right image size is not yet saved, see #5228. That likely needs updating.

That's a simple fix in the `handleChangeCover` callback in the `PublishPanel` component. Instead of using `image.sizes?.medium?` it would need to pick a different size, like `useLoadStory` is doing

**Comment by swissspidy** [comment]
The question is just whether you need the size of the original image, or the dimensions of the cropped image. I suspect the original.

**Comment by carloskelly13** [comment]
![Screen Shot 2020-11-24 at 2.54.18 PM.png](https://images.zenhubusercontent.com/5ea09bca6a44046f869bafe0/d4f296e4-339d-43b8-b4ee-8973e6753ea1)

Yup I think so! or what the API calls "full"

**Comment by zachhale** [comment]
Related to this: changing to use `full` size would only work for placed images, but for background images the image dimensions are not relevant in the `mediaElementResolution` checks we have and necessitate an additional implementation that calculates whether the resolution is sufficient when applied as a background image.

See comments in #5374.

**Comment by zachhale** [comment]
I think this is a similar issue to what's happening in #5377 and #5376 which I'm looking into. The resource `width` and `height` are getting set to values much smaller than the `full` size width and height even though the image URL for the resource is the the full size URL.
